<div>
   <h1 class="font-bold mb-3 text-xl">About Cov2Tree</h1>
   This website allows you to explore a phylogenetic tree with more than a million SARS-Cov-2 sequences, sequenced by researchers around the world. The interface was made by <a class="text-blue-700 underline" href="http://theo.io/">Theo Sanderson</a> using a custom-developed open-source library called <a href="http://github.com/theosanderson/taxonium" class="text-blue-700 underline"> Taxonium</a>. The Treenome Browser was developed by <a href="http://github.com/amkram" class="text-blue-700 underline">Alex Kramer</a>.
   <h1 class="font-bold mb-3 text-xl mt-3">The sequences</h1>
   <p>This tree displays sequences generated by scientists all over the world. The sequences are sourced from GenBank (i.e. the INSDC databases), the China National Center for Bioinformation, and from COG-UK. They are collated <a class="text-blue-700 underline" href="http://hgdownload.soe.ucsc.edu/goldenPath/wuhCor1/UShER_SARS-CoV-2//">here</a> by Angie Hinrichs and others at UCSC.</p>
   <p>This website would be empty but for the hard work of all these researchers, and their openness in submitting their work to public databases, and we gratefully acknowledge this.</p>
   <h1 class="font-bold mb-3 text-xl mt-3">The tree</h1>
   <p>The tree shown here is produced by the UCSC team, and made available <a class="text-blue-700 underline" href="http://hgdownload.soe.ucsc.edu/goldenPath/wuhCor1/UShER_SARS-CoV-2//">here</a>. It is derived from releases of Rob Lanfear's <a class="text-blue-700 underline" href="https://github.com/roblanf/sarscov2phylo">sarscov2phylo</a>, pruned to include only public sequences aggregated from GenBank, COG-UK, and the China National Center for Bioinformation. The tree has been re-rooted to Wuhan/Hu-1 (GenBank MN908947.3, RefSeq NC_045512.2), and nodes with no associated mutations have been collapsed. Sequences released after the final sarscov2phylo release (Nov. 13, 2020) have been added to the tree using <a class="text-blue-700 underline" href="https://github.com/yatisht/usher">UShER</a>, developed by Yatish Turakhia. The tree is maintained by Angie Hinrichs. UShER development and tree inference was done in Russ Corbett-Detig and David Haussler's groups at the UCSC Genomics Institute. Tree-building work is carried out by the UCSC team and described here only for completeness. They provide metadata with the sequences, including lineages called by <a class="text-blue-700 underline" href="https://www.pango.network/">Pangolin</a>.</p>
   <h1 class="font-bold mb-3 text-xl mt-3">Downloads</h1>
   <p>During the process of creating this tree we use <a  class="text-blue-700 underline" href="https://github.com/theosanderson/chronumental">Chronumental</a> to infer a time tree. For the convenience of others we are also making a download of this <a  class="text-blue-700 underline" href="https://cov2tree.nyc3.cdn.digitaloceanspaces.com/chron_tree.nwk.gz">tree</a> and the inferred <a  class="text-blue-700 underline" href="https://cov2tree.nyc3.cdn.digitaloceanspaces.com/chron_dates.tsv.gz">tip dates</a> available here.</p>
   <p class="mt-3">You can also download <a  class="text-blue-700 underline" href="https://cov2tree.nyc3.cdn.digitaloceanspaces.com/latest_public.jsonl.gz">the Taxonium JSONL file displayed here</a> or <a  class="text-blue-700 underline" href="https://taxonium.org/?protoUrl=https://cov2tree.nyc3.cdn.digitaloceanspaces.com/latest_public.jsonl.gz">load it in full-download mode</a> in Taxonium.
    <h1 class="font-bold mb-3 text-xl mt-3">Citations</h1>
   <div class="text-sm">
       <p>The tool: Sanderson (2022). <a class="text-blue-700 underline" href="https://elifesciences.org/articles/82392">Taxonium, a web-based tool for exploring large phylogenetic trees</a></p>
      <p>The tree: McBroome et al. (2020). <a class="text-blue-700 underline" href="https://www.biorxiv.org/content/10.1101/2021.04.03.438321v2/">A daily-updated database and tools for comprehensive SARS-CoV-2 mutation-annotated trees</a></p>
      <p>Treenome Browser: Kramer et al. (2023). <a class="text-blue-700 underline"  href="https://academic.oup.com/bioinformatics/article/39/1/btac772/6858450">Treenome Browser: co-visualization of enormous phylogenies and millions of genomes</a></p>
      <p><b>If referring to Cov2Tree please ensure you cite the underlying tree (McBroome et al.) as well as Taxonium.</b></p>
      
   </div>
   <h1 class="font-bold mb-3 text-xl mt-3 mb-1">Related tools</h1>
   <div class="text-sm">
      <p>You can also explore large phylogenies using <a class="text-blue-700 underline" href="https://microreact.org/">Microreact</a>, and sampled phylogenies on <a class="text-blue-700 underline" href="https://nextstrain.org/">Nextstrain</a>.</p>
   </div>
</div>
